### PR TITLE
Add checks for mask size and dimensions.

### PR DESCRIPTION
**Pull request by JoostJM**
Add function to check if ROI is large enough for computation. 
First checks if label is present in the mask, then checks if the number of dimensions in which the size > 1 is at least the minimum number of dimensions (default: 1). Finally an an optional check for a minimum number of voxels in the tumor region can be performed (switched off by default).

This check is performed in `featureextractor` after the loading of the image and any image pre-processing steps (e.g. resampling), but before features are extracted or filters applied.

Addresses #18 
cc @Radiomics/developers

### CODE REVIEW

File: bin/addClassToBaseline.py [hunk 1]
```diff
@@ -74,7 +74,8 @@ def main():
 
     provenance = extractor.getProvenance(imagePath, maskPath, mask)
 
-    image, mask, bb = imageoperations.cropToTumorMask(image, mask)
```
Comment (fedorov) [inline bin/addClassToBaseline.py]: I would avoid using "Tumor". Maybe "cropToMaskBoundingBox", or "cropToMaskROI".

Comment (fedorov) [inline bin/addClassToBaseline.py]: Sorry, I understand this is not related to the essence of this specific PR

**Comment by fedorov** [comment]
LGTM otherwise!

File: bin/addClassToBaseline.py (hunk 1 above)
Comment (JoostJM) [inline bin/addClassToBaseline.py]: > Sorry, I understand this is not related to the essence of this specific PR

No, it's not, but it's still a good suggestion. I am running over the documentation for errors, will update this then also.
